src/types/contributor.types.ts

```diff
@@ -21,11 +21,14 @@ export interface Contributor extends BackendType {
   sequence: number;
 }
 
+export interface Labels {
+  [key: string]: string;
+}
+
+// DOI lookup can give labels without id for institutions, while user will add IDs.
```

Comment:
hva betyr denne setningen? `while user will add IDs`?

DOI lookup kan gi institusjon uten id, men brukeren skal legge til id? skjønner ikke

Reply:
hm ja... prøver å formidle at når registeringen kommer rett fra DOI-oppslag kan contributors ha affiliations med labels (som er tekstinput vi ikke kan stole helt på).
men når man legger inn en ny contributor vil man legge til ID, og labels er ikke relevant.

er dette bedre?
"(...) while id will be used when user adds a contributor"

Reply:
```suggestion
// DOI lookup can give labels without id for institutions, while when the registration is added manually there will be ids present, and no need for labels.
```

Reply:
(bare contributor i stedet for registration da)